src/main/java/it/gov/pagopa/admissibility/utils/Utils.java

```diff
@@ -41,4 +46,43 @@ public static <T> T getHeaderValue(Message<?> message, String headerName) {
     public static Long euro2Cents(BigDecimal euro){
         return euro == null? null : euro.multiply(ONE_HUNDRED).longValue();
     }
+
+    //region birthdate from fiscalcode
+    public static LocalDate calculateBirthDateFromFiscalCode(String fiscalCode) {
+        // Extract birthdate characters from the fiscal code
+        String birthDateCode = fiscalCode.substring(6, 11);
```

Comment:
let's validate through regexp the fiscalCode structure, in order to be sure that it has the shape expected

Reply:
if not valid, we should throw an exception that should be handled as BirthDate KO (probably this logic is also in the PDND integrated branch...)